init.go

```diff
@@ -14,20 +17,47 @@ var initCmd = &cobra.Command{
 There must be a local git repository in order for initialization
 to succeed.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		err := client.InitializeInertiaProject()
+		err := client.InitializeInertiaProject(cmd.Parent().Version)
 		if err != nil {
 			log.Fatal(err)
 		}
-		println("A .inertia folder has been created to store Inertia configuration.")
-		println("It is recommended that you DO NOT commit this folder in source")
-		println("control since it will be used to store sensitive information.")
+		println("A .inertia.toml configuration file has been created to store")
+		println("Inertia configuration. It is recommended that you DO NOT commit")
+		println("this file in source control since it will be used to store")
+		println("sensitive information.")
 		println("\nYou can now use 'inertia remote add' to connect your remote")
 		println("VPS instance.")
 	},
 }
 
+var resetCmd = &cobra.Command{
+	Use:   "reset",
+	Short: "Reset the Inertia project in this repository.",
+	Long: `Reset removes the existing Inertia configuration from
+	this repository.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		println("WARNING: This will remove your current Inertia configuration")
+		println("and is irreversible. Continue? (y/n)")
+		var response string
+		_, err := fmt.Scanln(&response)
+		if err != nil {
+			log.Fatal("invalid response - aborting")
+		}
+		if response != "y" {
```

Comment:
Is it standard behaviour for cli tools to treat all inputs except `"y"` as a `"n"`?

Reply:
I am not too sure actually. Personally I think since this is a destructive action, any command other than `y` can safely be assumed to be a "no"

Reply:
Ok, that's fine with me then!